### PR TITLE
Scavengers Twist: Scavengers Feud

### PR DESCRIPTION
**Pull request by sparkyneko**
Should be approved by room owners nyaa

### CODE REVIEW

File: server/chat-plugins/scavenger-games.ts [hunk 1]
```diff
@@ -361,6 +363,81 @@ const TWISTS: {[k: string]: Twist} = {
...
 		},
 	},
+
+	mistakes: {
+		id: 'mistakes',
+		name: 'Mistakes',
+		desc: 'After completing the hunt, players will guess what the most common incorrect answer for each question is.',
+		onAfterLoad() {
+			this.guesses = {};
+			this.incorrect = new Array(this.questions.length).fill('').map(() => ({}));
```
Comment (AnnikaCodes) [inline server/chat-plugins/scavenger-games.ts:373]:
```suggestion
			this.incorrect = new Array(this.questions.length).fill({});
```

**Comment by aQrator** [comment]
Not sure if you still need this but uh: approved

**Comment by aQrator** [comment]
@AnnikaCodes Hi could this get merged if it's approved please?

**Comment by AnnikaCodes** [comment]
I don't know what this is or if it will break anything...

File: server/chat-plugins/scavenger-games.ts (hunk 1 above)
Comment (sparkyneko) [reply server/chat-plugins/scavenger-games.ts]: Oh I remember why I did this originally like this - .fill({}) filled it with references to the same object which doesn't work.  Gotta quickly PR and revert it back.

**Pull request merged**